src/lucky_migrator/column_default_helpers.cr

```diff
@@ -1,28 +1,36 @@
 module LuckyMigrator::ColumnDefaultHelpers
   alias ColumnDefaultType = String | Time | Int32 | Int64 | Float32 | Float64 | Bool | Symbol
```

Comment:
Maybe this could be just `ColumnType` since it is used outside of defaults now?

Reply:
I saw that too but it conflicts with the `ColumnType` alias in `ColumnTypeOptionHelpers`

Reply:
Do you think I should change the existing `ColumnType` to something else like `ColumnTypeClass` and rename `ColumnDefaultType` to `ColumnType`?

Reply:
I think this is ok for now 👍